Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -13,6 +13,8 @@ protocol PhotonActionSheetProtocol {
 
 extension PhotonActionSheetProtocol {
     typealias PresentableVC = UIViewController & UIPopoverPresentationControllerDelegate
+    typealias MenuAction = () -> Void
+    typealias URLOpenAction = (URL?, Bool) -> Void
```

Comment:
We should consider an enum in the project for TabPrivacy.private/.public. 
I had to dig a bit to see what this bool was for.

Reply:
Not a bad idea. I think I can add names to this though. I'll do that instead

Reply:
Cant add names. I created a new typealias to Bool